docs/types/value-types.rst

```diff
@@ -161,19 +161,33 @@ Fixed Point Numbers
     cannot be assigned to or from.
 
 ``fixed`` / ``ufixed``: Signed and unsigned fixed point number of various sizes. Keywords ``ufixedMxN`` and ``fixedMxN``, where ``M`` represents the number of bits taken by
-the type and ``N`` represents how many decimal points are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
+the type and ``N`` represents how many fractional decimal digits are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
 ``ufixed`` and ``fixed`` are aliases for ``ufixed128x18`` and ``fixed128x18``, respectively.
 
+Fixed point types are used to represent fractional numbers with a fixed precision. The number ``1.5`` is stored in a ``fixed8x2`` (8 bit number with
+two fractional digits) as ``1.50``. In the EVM, it is stored exactly the same way as the number ``150`` would be stored in an ``int8``.
+The largest and the smallest number representable in a ``fixed8x2`` are ``1.27`` and ``-1.28``, respectively.
```

Comment:
Oh, so it's possible for `N` to be larger than the maximum number of significant digits? I.e. something like `fixed8x5` is still valid and `123` stored in it would mean `0.00123` and the type would represent numbers between `-0.00128` and `0.00127`? That's nice. Might be worth pointing it out (at least saying that such types are valid).

Reply:
I think a good condition is that `1` has to be in the value range, but I'm actually not sure what we chose in the end.